src/modules/components/proposal/proposalVoting/proposalVoting.stories.tsx

```diff
@@ -197,22 +238,173 @@ export const SingleStage: Story = {
                     startDate={DateTime.now().toMillis()}
                     endDate={DateTime.now().plus({ hours: 7 }).toMillis()}
                 >
-                    <ProposalVoting.BreakdownMultisig
-                        approvalsAmount={multisigVotes.length}
-                        minApprovals={minApprovals}
-                    />
-                    <ProposalVoting.Votes>
-                        <DataList.Root itemsCount={filteredVotes.length} entityLabel="Votes">
-                            <DataList.Filter searchValue={search} onSearchValueChange={setSearch} />
-                            <DataList.Container>
-                                {filteredVotes.map((vote) => (
-                                    <VoteDataListItem.Structure key={vote.voter.address} {...vote} />
-                                ))}
-                            </DataList.Container>
-                            <DataList.Pagination />
-                        </DataList.Root>
-                    </ProposalVoting.Votes>
-                    <ProposalVoting.Details settings={getMultisigSettings(minApprovals)} />
+                    <ProposalVoting.BodyContent status={ProposalVotingStatus.ACTIVE}>
+                        <FoundersApprovalContent
+                            multisigSearch={search}
+                            setMultisigSearch={setSearch}
+                            minApprovals={minApprovals}
+                        />
+                    </ProposalVoting.BodyContent>
+                </ProposalVoting.Stage>
+            </ProposalVoting.Container>
+        );
+    },
+};
+
+/**
+ * Usage example of the ProposalVoting module component for single stage with multi body
+ */
+export const SingleStageMultiBody: Story = {
+    args: {
+        className: 'max-w-[560px]',
+        title: 'Proposal Voting',
```

Comment:
The components doesn't handle the `title` props anymore, we can remove this. Same for the story below